packages/react-scripts/scripts/test.js

```diff
@@ -44,3 +44,8 @@ argv.push(
 );
 // @remove-on-eject-end
 jest.run(argv);
+
+process.stdin.on('end', function() {
```

Comment:
We need to only do this in non-CI mode. Move this into `if` statement below plz.